lib/copy/ncp.js

```diff
@@ -96,7 +96,7 @@ function ncp (source, dest, options, callback) {
 
   function copyFile (file, target) {
     var readStream = fs.createReadStream(file.name)
-    var writeStream = fs.createWriteStream(target, { mode: file.mode })
+    var writeStream = fs.createWriteStream(target)
```

Comment:
Interesting, I'm not sure either. Wouldn't the default permission be such that you'd still be able to do anything to it after writing?

A similar module [cpr](https://github.com/davglass/cpr) has been doing it without it (there's still a [feature-req](https://github.com/davglass/cpr/issues/34) for it).